client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1122,6 +1123,26 @@ static Request analyze(AnalyzeRequest request) throws IOException {
         return req;
     }
 
+    static Request termVectors(TermVectorsRequest tvrequest) throws IOException {
+        String endpoint = "_termvectors";
+        if (tvrequest.getId() != null) {
```

Comment:
the EndpointBuilder() gladly accepts nulls in its `addPathPart(...)` method. You dont need to worry about passing a null value in, as it just wont put it in the path.